core/types/withdrawal.go

```diff
@@ -42,6 +43,10 @@ type withdrawalMarshaling struct {
 	Amount    hexutil.Uint64
 }
 
+func (w Withdrawal) Size() common.StorageSize {
+	return common.StorageSize(unsafe.Sizeof(w))
```

Comment:
Please avoid the use of package unsafe here. Since withdrawals are defined by the protocol and will never change, we can just hard-code the returned size here.